src/common-models/shift-info.model.ts

```diff
@@ -13,8 +13,7 @@ export enum ShiftCode {
 }
 
 export interface ShiftInfoModel {
-  // eslint-disable-next-line @typescript-eslint/camelcase
```

Comment:
Proponuję lub usunąć tą regułę, lub zaimplementować mapowanie.

Reply:
Wyłączamy globalnie i dodajemy taska na mapowanie.

Reply:
Dodałem task na Tarze. Próbowałem też wyłączyć tą opcję, ale po 20 minutach nieudanych prób uznałem, że to nie jest aż takie ważne skoro i tak będziemy robić mapowanie modelu.